app/controllers/sessions_controller.rb

```diff
@@ -2,9 +2,12 @@ class SessionsController < ApplicationController
   skip_before_action :authenticate_user!, :set_organization, :authorize_organization_access
 
   def new
-    redirect_to '/auth/github'
+    scopes = session[:required_scopes] || 'user:email,repo,delete_repo,admin:org'
```

Comment:
@mastahyeti Maybe we should put these scopes behind a method like `default_scopes` or `all_scopes` to make this a bit more clear.

There shouldn't be cases where `session[:required_scopes]` isn't set, so this seems to be just a backup for safety.